database/migrations/20221102141110-create-user.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstName: {
+        type: Sequelize.STRING
+      },
+      lastName: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.STRING
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      avatar: {
+        type: Sequelize.STRING
+      },
+      roleId: { 
+        type: Sequelize.INTEGER, 
+        references: { model: { tableName: 'Role', schema: 'schema' }, key: 'id' },
```

Comment:
### Pendingn

Code correct:

`roleId: { 
        type: Sequelize.INTEGER, 
        references: { 
          model: 'Roles' , 
          key: 'id' 
        },
        allowNull: false 
      },`

View in mysql Workbench:

![Captura de Pantalla 2022-11-03 a la(s) 2 39 37 p m](https://user-images.githubusercontent.com/84211273/199818284-d8743b36-3624-4f16-a3be-c0714bddb9ef.png)